source/installing-splunk/splunk-app.rst

```diff
@@ -67,4 +67,66 @@ Installation
     :align: center
     :width: 100%
 
-Now that you've finished installing Splunk app for Wazuh, you can install and setup Splunk forwarders on the :ref:`next section <splunk_forwarder>`.
+Now that you've finished installing Splunk app for Wazuh, you can install and setup Splunk forwarders on the :ref:`Splunk forwarder section <splunk_forwarder>`.
+
+Installing the Wazuh App in a Splunk cluster
+--------------------------------------------
+
+.. note::
+  We can install the App in each search-head by hand, but if we have hundreds or even thousands of search-heads, it will be better to install it automatically.
+  For this purpose, we are using the **deployer**, a machine that installs the App in every search-head at the same time and automatically.
+
+After installing the App following the **Official installation guide** in our **deployer** machine, we follow this steps:
+
+.. code-block:: console
+
+  // Copy the app into the splunk cluster folder:
+  # cp -r installation_path/SplunkAppForWazuh /opt/splunk/etc/shcluster/apps
+
+  // Create the file that listens the outputs from the Wazuh API:
+  # touch /opt/splunk/etc/shcluster/apps/SplunkAppForWazuh/default/outputs.conf
+
+Then, we fill the outputs.conf file wit the next lines:
+
+.. code-block:: xml
+
+  [indexer_discovery:cluster1]
+  pass4SymmKey = changeme
+  master_uri = https://<master_ip>:<management_port>
+
+  [tcpout:cluster1_tcp]
+  indexerDiscovery = cluster1
+
+  [tcpout]
+  defaultGroup = cluster1_tcp
+
+.. note::
+  We use indexerDiscovery to connect to peer nodes. Click `here <https://docs.splunk.com/Documentation/Splunk/7.1.3/Indexer/indexerdiscovery>`_ to check more info about indexerDiscovery.
+
+.. note::
+  <master_ip> references to the search-heads master ip.
```

Comment:
The `Indexer` master, not the search head master.

Reply:
Also, explain that the `https` is required by default and also that the default port is the `8089`.